uint32 src_vehicle
uint32 dst_vehicle
uint32 platoon_id